docs/en/stack/ml/nlp/ml-nlp-ner-example.asciidoc

```diff
@@ -0,0 +1,309 @@
+[[ml-nlp-ner-example]]
+= How to deploy named entity recognition (NER)
```

Comment:
IMO you only have to define this acronym once and the initial paragraph is a better place to do that (which you already have there)

```suggestion
= How to deploy named entity recognition
```